server/statistics/store_hot_peers_infos.go

```diff
@@ -32,6 +32,41 @@ type StoreHotPeersInfos struct {
 // NOTE: This type is exported by HTTP API. Please pay more attention when modifying it.
 type StoreHotPeersStat map[uint64]*HotPeersStat
 
+// CollectHotPeerInfos only returns TotalBytesRate,TotalKeysRate,TotalQueryRate,Count
+func CollectHotPeerInfos(stores []*core.StoreInfo, regionStats map[uint64][]*HotPeerStat) *StoreHotPeersInfos {
+	peerLoadSum := make([]float64, DimLen)
+	collect := func(kind core.ResourceKind) StoreHotPeersStat {
+		ret := make(StoreHotPeersStat, len(stores))
+		for _, store := range stores {
+			id := store.GetID()
+			hotPeers, ok := regionStats[id]
+			if !ok {
+				continue
+			}
+			for i := range peerLoadSum {
+				peerLoadSum[i] = 0
+			}
+			peers := filterHotPeers(kind, hotPeers)
+			for _, peer := range peers {
+				for j := range peerLoadSum {
+					peerLoadSum[j] += peer.GetLoad(j)
+				}
+			}
+			ret[id] = &HotPeersStat{
```

Comment:
Will it miss some info compared with before?

Reply:
It doesn't contain store info and hot peers detail, but metrics only use the total data of hot peers.